apex_launchtest/package.xml

```diff
@@ -1,19 +1,18 @@
 <?xml version="1.0"?>
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="2">
-  <name>apex_rostest</name>
+  <name>apex_launchtest</name>
   <version>0.2.0</version>
   <description>A package providing a ROS2 integration test framework.</description>
   <maintainer email="pete.baughman@apex.ai">Pete Baughman</maintainer>
   <license>Apex.AI Proprietary License</license>
```

Comment:
Apex.AI Proprietary License => Apache 2.0

This should be changed in all 3 package.xml files

Reply:
@dejanpan done